src/tsickle.ts

```diff
@@ -888,6 +888,36 @@ class ExternsWriter extends ClosureRewriter {
       this.writeExternsFunction(name.getText(), paramNames, namespace);
     }
 
+    // Handle method declarations/signatures first, since we need to deal with overloads.
+    // TODO: Build a collection of the remaining members
+    let methods: {[id: string]: ts.MethodDeclaration[];} = {};
+    for (let member of decl.members) {
+      if (member.kind === ts.SyntaxKind.MethodSignature ||
+          member.kind === ts.SyntaxKind.MethodDeclaration) {
+        let m = <ts.MethodDeclaration>member;
+        let overloaded = methods[m.name.getText()];
```

Comment:
Ignoring this comment per your later one. I do need the method name here since it is the unique key for the map of 1...n methods variants I'm building.